src/app/policies/components/PolicyList.vue

```diff
@@ -0,0 +1,288 @@
+                  class="actions-dropdown"
+                  :kpop-attributes="{ placement: 'bottomEnd', popoverClasses: 'mt-5 more-actions-popover' }"
+                  width="150"
+                >
+                  <template #default>
+                    <KButton
+                      class="non-visual-button"
+                      appearance="secondary"
+                      size="small"
+                    >
+                      <template #icon>
+                        <KIcon
+                          color="var(--black-400)"
+                          icon="more"
+                          size="16"
+                        />
+                      </template>
+                    </KButton>
+                  </template>
+
+                  <template #items>
+                    <KDropdownItem
+                      :item="{
+                        to: {
+                          name: 'policy-detail-view',
+                          params: {
+                            mesh: route.params.mesh,
+                            policyPath: policyType.path,
+                            policy: row.name,
+                          },
+                        },
+                        label: t('common.collection.actions.view'),
+                      }"
+                    />
+                  </template>
+                </KDropdownMenu>
+              </template>
+            </AppCollection>
+          </template>
+        </KCard>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import {
+  KBadge,
+  KButton,
+  KCard,
+  KDropdownItem,
+  KDropdownMenu,
+  KIcon,
+} from '@kong/kongponents'
+import { computed } from 'vue'
+import { useRoute } from 'vue-router'
+
+import { PolicyCollection } from '../sources'
+import AppCollection from '@/app/application/components/app-collection/AppCollection.vue'
+import DocumentationLink from '@/app/common/DocumentationLink.vue'
+import PolicyTypeTag from '@/app/common/PolicyTypeTag.vue'
+import type { MeshInsight, PolicyType } from '@/types/index.d'
+import { useI18n } from '@/utilities'
+import { notEmpty } from '@/utilities/notEmpty'
+
+type ChangeValue = {
+  page: number
+  size: number
+  s: string
+}
+
+const { t } = useI18n()
+const route = useRoute()
+
+const props = defineProps<{
+  pageNumber: number
+  pageSize: number
+  policyTypes: PolicyType[]
+  currentPolicyTypePath: string
+  policyCollection: PolicyCollection | undefined
+  policyError: Error | undefined
+  meshInsight: MeshInsight | undefined
+}>()
+
+const emit = defineEmits<{
+  (event: 'change', value: ChangeValue): void
+}>()
+
+const policyType = computed(() => props.policyTypes.find((policyType) => policyType.path === props.currentPolicyTypePath) ?? props.policyTypes[0])
+
+const sortedPolicyTypeWrappers = computed(() => props.policyTypes.map((policyType) => ({
+  policyType,
+  numberOfPolicies: props.meshInsight?.policies?.[policyType.name]?.total ?? 0,
+})))
```

Comment:
AS this computed prop is only used once I'd inline it above.  like commented previously

Reply:
Done.